samples/web/content/apprtc/js/sdputils.js

```diff
@@ -8,7 +8,7 @@
 
 /* More information about these options at jshint.com/docs/options */
 
-/* globals displayError, params */
+/* globals displayError */
```

Comment:
Not introduced by this PR, but there is no displayError any more. Could you change it to trace?